components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/listener/IdentityOathEventListener.java

```diff
@@ -372,6 +372,8 @@ private void removeTokensFromCache(String userName, UserStoreManager userStoreMa
         authenticatedUser.setTenantDomain(tenantDomain);
         authenticatedUser.setUserName(userName);
         try {
+            // Only the tokens and auth codes issued for openid scope should be removed from the cache, since no
+            // claims are usually cached against tokens or auth codes, otherwise.
```

Comment:
Shall we make this a multi-line comment?

Reply:
done